CHANGES.rst

```diff
@@ -65,6 +65,7 @@ Improvements
   reviewing
 - Show a nicer error message when entering an excessively high base
   registration fee (:issue:`3260`)
+- Add the keywords as meta tag to the page header(thanks :user:`bpedersen2`)
```

Comment:
Missing space, and IMHO this can be a bit more concise:

```markdown
- Expose event keywords through an HTML meta tag (:issue:`3262`, thanks
  :user:`bpedersen2`)
```

Reply:
Done